doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-mortgages-example-proc.adoc

```diff
@@ -18,51 +18,67 @@ The package you select must contain all the required rule assets.
 . Click *Ok* to create and open the test scenario in the test scenarios designer.
 . Expand *Project Explorer* and verify the following:
 * `Applicant`, `Bankruptcy`, `IncomeSource`, and `LoanApplication` data objects exist.
-* `Pricing loans` guide decision table exists.
+* `Pricing loans` guided decision table exists.
 * Verify that the new test scenario is listed under *Test Scenario*
 . After verifying that everything is in place, return to the *Model* tab of the test scenarios designer and define the *GIVEN* and *EXPECT* data for the scenario, based on the available data objects.
 +
 .A blank test scenarios designer
-image:project-data/test-scenarios-preview-editor.png[]
+image:project-data/test-scenarios-blank-preview-editor.png[]
 +
 . Define the *GIVEN* column details,
 .. Click the cell named *INSTANCE 1* under the *GIVEN* column header.
 .. From *Test Tools* panel, select *LoanApplication* data object.
```

Comment:
Minor suggestion - Add the article:

s/From *Test Tools* panel, select *LoanApplication* data object./From the *Test Tools* panel, select the *LoanApplication* data object.

Reply:
Done